arangod/Cache/Table.cpp

```diff
@@ -43,6 +43,16 @@ void Table::GenericBucket::unlock() {
   _state.unlock();
 }
 
+void Table::GenericBucket::clear() {
+  _state.lock(UINT64_MAX, [this]() -> void {
+    _state.clear(); // "clear" will keep the lock!
+    for (size_t i = 0; i < BUCKET_SIZE - sizeof(BucketState); ++i) {
+      _filler[i] = 0;
```

Comment:
Is this necessary? Seems to me like `_filler` is only defined to ensure that `sizeof(GenericBucket)` equals `BUCKET_SIZE`, but it is never actually used.